project-service/src/middleware/DiscoveryServer.js

```diff
@@ -0,0 +1,38 @@
+const axios = require('axios').default;
+const logger = require('../config/logger/logger')
+const eventEmitter = require('../config/events/events');
+
+let serverList = []
+let syncStatus = false
+
+const discoveryServer = async () => {
+
+    eventEmitter.on('eureka-hartbeat-successfull', async () => {
+        const eurekaHost = `${process.env.SSL ? 'https' : 'http'}://${process.env.EUREKA_HOST || 'localhost:8761'}/eureka/apps/`;
```

Comment:
Configuration should be externalised. (eg : "Eureka Host"). Best would be to use a YAML file. Netflix gets configuration from configuration server. So follow the same Spring boot pattern for getting config.

https://www.npmjs.com/package/config